Mathlib/Tactic/Linter/Style.lean

```diff
@@ -68,4 +69,47 @@ initialize addLinter setOptionLinter
 
 end Style.setOption
 
+/-- The `check_declID` linter: if it emits a warning, then a declID is considered
```

Comment:
How about this?
The `check_declID` linter emits a warning on declarations whose name is non-standard style.
(Currently, this only includes declarations whose names include a double underscore.)